src/OpenSage.Game/Data/Ini/Parser/IniParser.Attributes.cs

```diff
@@ -37,7 +37,7 @@ public string ParseAttributeIdentifier(string label)
         }
 
         public T ParseAttributeEnum<T>(string label)
-            where T : struct
+            where T : struct, IConvertible
```

Comment:
Isn’t there an Enum generic constraint now?

Reply:
Oh that would be awesome. Didn't know that this existed. Will change.

Reply:
Can't use that Enum constraint, because it doesn't work with the BitArray<T> stuff. Enum is a nullable type. Combining "Enum, struct" is not sufficient either.

Reply:
Would it work if you changed the constraints on BitArray as well?